python/ray/tests/py3_test.py

```diff
@@ -247,3 +247,17 @@ def throw_error(self):
 
     with pytest.raises(ray.exceptions.RayTaskError):
         await ray.async_compat.get_async(direct.throw_error.remote())
+
+
+def test_asyncio_actor_async_get(ray_start_regular_shared):
```

Comment:
Is this the only test for asyncio actors right now? If so, could you add some more tests?